docs/markdown/Syntax.md

```diff
@@ -186,6 +186,17 @@ As can be seen, the formatting works by replacing placeholders of type
 Strings also support a number of other methods that return transformed
 copies.
 
+#### .replace()
+
+Since 0.58.0, you can now replace characters in a string
+
+```meson
+# Similar to the Python str.replace(). Replaces instances of one character with another
```

Comment:
I think we voluntarily avoid documenting stuff as reference to python, because it's really an implementation detail.

Reply:
I would also specify "replaces **all** instances". (I hate how this doc is duplicated)